### PR TITLE
LV translation updates for v4.1.1

### PR DESCRIPTION
**Pull request by davislasis**
LV language translation updates for v4.1.1
- new files: Cookie and Security
- language keys merged to perfectly match EN version
- each language key has original EN translation as comment
- updated test
- updated README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ claims no expertise beyond English.
 | ja     | Japanese             | 4.0.3   | [kenjis][ja]         | Kenji Suzuki
 | ko     | Korean               | 4.0.2   | [codeigniter-kr][kr] | codeigniter-kr
 | lt     | Lithuanian           | 4.0.2   | [dgvirtual][lt]      | Donatas Glodenis
-| lv     | Latvian              | 4.0.3   | [davislasis][lv]     | Dāvis Lasis
+| lv     | Latvian              | 4.1.1   | [davislasis][lv]     | Dāvis Lasis
```
Comment (paulbalandan) [inline README.md:47]: Please revert this change. LV is introduced in 4.0.3

**Review by paulbalandan — CHANGES_REQUESTED** [review]
Please run `vendor/bin/php-cs-fixer fix -v` to fix the code style violations.

File: README.md (hunk 1 above)
Comment (davislasis) [reply README.md:47]: ok, i thought it is "in what current state this language translation is in".
I must delay this commit / fix because of problems getting php-cs-fixer in windows machine. I hope in next week or so i will get a new linux server up and running and then i can re-commit.

Thanks

**Comment by davislasis** [comment]
@paulbalandan 
changes made, php-cs-fixer done - ready to review and merge

Thanks

File: .github/scripts/continuous-integration [hunk 2]
```diff
@@ -24,8 +24,7 @@ if ($event === '--pull-request')
 {
 	$files = shell_exec('git diff --name-only --diff-filter=ACMRTUXB origin/develop..HEAD');
 }
-else
-{
+else {
```
Comment (paulbalandan) [inline .github/scripts/continuous-integration:27]: Please revert the changes made to this file. I think the fix is a bug in php-cs-fixer.

Comment (davislasis) [reply .github/scripts/continuous-integration:27]: ok, but it will fail tests

Comment (paulbalandan) [reply .github/scripts/continuous-integration]: Yes, that's the compromise for now.

**Review by paulbalandan — APPROVED** [review]
